app/models/ChannelTest.scala

```diff
@@ -43,6 +45,8 @@ trait ChannelTest[T] {
   def withPriority(priority: Int): T
 }
 
-trait ChannelTests[T <: ChannelTest[T]] {
-  val tests: List[T]
+case class ChannelTests[T : Decoder : Encoder](tests: List[T])
```

Comment:
the client still expects this model in the response, but there's no longer any need to inherit from `ChannelTests` on the backend